ui/page/main_page.go

```diff
@@ -470,7 +475,32 @@ func (mp *MainPage) LayoutUSDBalance(gtx layout.Context) layout.Dimensions {
 	)
 }
 
+func (mp *MainPage) HideMyBalance(gtx layout.Context) layout.Dimensions {
```

Comment:
This is not used outside this file. 
```suggestion
func (mp *MainPage) hideMyBalance(gtx layout.Context) layout.Dimensions {
```